common/errors/code.go

```diff
@@ -16,4 +16,5 @@ const (
 	ServiceDoesNotExist      ErrorCode = "ServiceDoesNotExist"
 	TaskDoesNotExist         ErrorCode = "TaskDoesNotExist"
 	UnexpectedError          ErrorCode = "UnexpectedError"
+	TimeOut                  ErrorCode = "TimeOut"
```

Comment:
[nit] I'm a little bit on the fence here; a `Timeout` error code does accurately reflect what went wrong, but, given we are going to have the `client` retry any timeout it encounters, I'm worried someone may use this error code on a function that should not be retried? Maybe I'm being overly cautious? Curious what you think.